tasks/main.yml

```diff
@@ -5,6 +5,13 @@
   include: floating_ip.yml
   when: (COMMON_FLOATING_INTERFACE_NAME and COMMON_FLOATING_IP)
 
+- name: Add fallback DNS servers to resolve.conf.
```

Comment:
@smarnach Unless there's a reason not to (?), can we make this match the target file (`/etc/dhcp/dhclient.conf`)?